version.go

```diff
@@ -1,3 +1,3 @@
 package main
 
-const VERSION = "1.19.2"
```

Comment:
I'd say this deserves a major version bump

Reply:
should be have 2.0.0 or 1.20.0 ?

Reply:
Just a refresher! [MAJOR.MINOR.PATCH](https://semver.org/)